services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/internal/GenericS3TransferManager.java

```diff
@@ -457,9 +472,17 @@ public final Copy copy(CopyRequest copyRequest) {
 
         CompletableFuture<CompletedCopy> returnFuture = new CompletableFuture<>();
 
-        TransferProgressUpdater progressUpdater = new TransferProgressUpdater(copyRequest, null);
-        progressUpdater.transferInitiated();
-        progressUpdater.registerCompletion(returnFuture);
+        // set length to 10000 as reference value, since we don't make HeadObject call yet
+        TransferProgressUpdater progressUpdater = new TransferProgressUpdater(copyRequest, 10000L);
+        if (isS3ClientMultipartEnabled()) {
```

Comment:
Do we only want to enable progress listening if it is a multi-part request ? Looks like the behaviour previously called progress updates regardless

Reply:
This is basically a check for Java-based vs CRT-based client. Supporting this for CRT would require additional work on the CRT side. Currently INFO logs print `"TransferInitiated"` and nothing after that which isn't ideal. Think its better to just not print anything at all, wdyt?

Reply:
I see, so it is nothing meaningful. Got it, this way is better. Would be great to see that in a comment though